test_harness_inputs/generated-platform-cert-test-list.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
should we add some form of "tag/sha/other" to reference the origin of the test plan repo this originates from? Or even a date?

That way we know how recent this is (more than looking at git log dates).